spec/functions/number.md

```diff
@@ -526,6 +490,117 @@ together with the resolved options' values.
+>
+> ```
+> The total was {0.5 :percent}.
+> ```
+>
+> should format in a manner similar to:
+>
+> > The total was 50%.
+
+#### Options
+
+Some options do not have default values defined in this specification.
+The defaults for these options are implementation-dependent.
+In general, the default values for such options depend on the locale,
+the value of other options, or both.
+
+> [!NOTE]
+> The names of _options_ and their _values_ were derived from the
+> [options](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#options)
+> in JavaScript's `Intl.NumberFormat`.
+
+The following options and their values are required to be available on the function `:number`:
+
+- `numberingSystem`
+  - valid [Unicode Number System Identifier](https://cldr-smoke.unicode.org/spec/main/ldml/tr35.html#UnicodeNumberSystemIdentifier)
+    (default is locale-specific)
+- `signDisplay`
+  - `auto` (default)
+  - `always`
+  - `exceptZero`
+  - `negative`
+  - `never`
+- `useGrouping`
+  - `auto` (default)
+  - `always`
+  - `never`
+  - `min2`
+- `minimumIntegerDigits`
+  - ([digit size option](#digit-size-options), default: `1`)
+- `minimumFractionDigits`
+  - ([digit size option](#digit-size-options))
+- `maximumFractionDigits`
+  - ([digit size option](#digit-size-options))
+- `minimumSignificantDigits`
+  - ([digit size option](#digit-size-options))
+- `maximumSignificantDigits`
+  - ([digit size option](#digit-size-options))
+- `trailingZeroDisplay`
+  - `auto` (default)
+  - `stripIfInteger`
+- `roundingPriority`
+  - `auto` (default)
+  - `morePrecision`
+  - `lessPrecision`
+- `roundingIncrement`
+  - 1 (default), 2, 5, 10, 20, 25, 50, 100, 200, 250, 500, 1000, 2000, 2500, and 5000
+- `roundingMode`
+  - `ceil`
+  - `floor`
+  - `expand`
+  - `trunc`
+  - `halfCeil`
+  - `halfFloor`
+  - `halfExpand` (default)
+  - `halfTrunc`
+  - `halfEven`
+
+If the _operand_ of the _expression_ is an implementation-defined type,
+such as the _resolved value_ of an _expression_ with a `:number` or `:integer` _annotation_,
+it can include option values.
+In general, these are included in the resolved option values of the _expression_,
+with _options_ on the _expression_ taking priority over any option values of the _operand_.
+Option values with the following names are however discarded if included in the _operand_:
+
+- `compactDisplay`
+- `notation`
+- `select`
+
+> For example, the _placeholder_ in this _message_:
+>
+> ```
+> .input {$n :number minimumFractionDigits=2 roundingMode=floor}
+> {{{$n :percent minimumFractionDigits=1}}}
+> ```
+>
+> would be formatted with the resolved options
+> `{ minimumFractionDigits: '1', roundingMode: 'floor' }`.
+
+#### Resolved Value
+
+The _resolved value_ of an _expression_ with a `:percent` _function_
+contains an implementation-defined numerical value
+of the _operand_ of the annotated _expression_ multiplied by 100,
+together with the resolved options' values.
```

Comment:
For clarity, and since "resolved value" and "options" are the terms, perhaps:

```suggestion
together with the _resolved values_ of the _options_.
```

Reply:
Variants of this same snippet are used in nearly all the functions. This change would be better made separately, affecting all of them at once.

Reply:
Perhaps "_resolved values_ of the resolved list of _options_"? Oh the pedantry...